assets/cubyz/biomes/ocean/cliffs.zig.zon

```diff
@@ -0,0 +1,38 @@
+.{
+	.properties = .{
+		.ocean,
+		.wet,
+		.mountain,
+	},
+	.radius = 128,
+	.minHeight = -1,
+	.maxHeight = 5,
+	.chance = 0.25,
+	.mountains = 50,
```

Comment:
The cliffs that stick out of the water do appear rather flat/round. Maybe more mountain noise would be good?